README.md

```diff
@@ -217,43 +175,72 @@ func application(application: UIApplication, openURL url: NSURL, sourceApplicati
+**isReferrable** (BOOL) _optional_
+: This boolean lets you control whether or not the user is eligible to be 'referred'. This is applicable for credits and influencer tracking. If isReferrable is set to NO | false, and the user clicks a link before entering the app, deep link parameters will appear, but that user will _not_ be considered referred. If isReferrable is set to YES | true, and the user clicks a link, deep link params will appear and the user _will_ be considered referred. Remove this argument to access the default, which only allows the user to be referred on a _fresh install_, but not on opens.
 
-#### Retrieve session (install or open) parameters
+###### handleDeepLink
 
-These session parameters will be available at any point later on with this command. If no params, the dictionary will be empty. This refreshes with every new session (app installs AND app opens)
+**url** (NSString *) _required_
+: This argument passes us the URI string so that we can parse the extra parameters. For example, 'myapp://open?link_click_id=12345'.
+
+#### Returns
+
+###### initSession
+
+Nothing
+
+###### handleDeepLink
+
+**BOOL** handleDeepLink will return a boolean indicating whether Branch has handled the URI. If the URI call is 'myapp://open?link_click_id=12345', then handleDeepLink will return YES because the Branch click object is present. If just 'myapp://', handleDeepLink will return NO.
+
+###Retrieve session (install or open) parameters
+
+These session parameters will be available at any point later on with this command. If no params, the dictionary will be empty. This refreshes with every new session (app installs AND app opens).
+
+####Methods
 
 ###### Objective-C
 
 ```objc
-NSDictionary *sessionParams = [[Branch getInstance] getLatestReferringParams]; // previously getReferringParams
+NSDictionary *sessionParams = [[Branch getInstance] getLatestReferringParams];
 ```
 
 ###### Swift
 
 ```swift
-let sessionParams = Branch.getInstance().getLatestReferringParams() // previously getReferringParams()
+let sessionParams = Branch.getInstance().getLatestReferringParams()
 ```
 
-#### Retrieve install (install only) parameters
+####Parameters
+
+None
 
-If you ever want to access the original session params (the parameters passed in for the first install event only), you can use this line. This is useful if you only want to reward users who newly installed the app from a referral link or something.
+####Returns
+
+**NSDictionary *** When initSession returns a parameter set in the deep link callback, we store it in NSUserDefaults for the duration of the session in case you want to retrieve it later. Careful, once the app is minimized and the session ends, this will be cleared.
+
+###Retrieve Install (Install Only) Parameters
```

Comment:
These are updated in other instances, fwiw. When you setIdentity, for instance, if that identity already existed on the backend, the install params for that identity will be set within the app.